semantic-java/src/Language/Java/Tags.hs

```diff
@@ -140,7 +140,7 @@ instance ToTags Java.FieldAccess
 instance ToTags Java.FieldDeclaration
 instance ToTags Java.FinallyClause
 instance ToTags Java.FloatingPointType
-instance ToTags Java.ForInit
+-- instance ToTags Java.ForInit
```

Comment:
Got errors on `ForInit` and `LocalVariableDeclarationStatement` so commented out for now, will return to this.

Reply:
I’m guessing the grammar was updated, is all.